### DISCUSSION TITLE
As a new user it would be useful if the landing pages for the website / project contained some more descriptive information.

### DISCUSSION THREAD

**Discussion by butler54** [fybrik/fybrik#588] [Ideas]
The current website is a little awkward. There is an about page which is the landing zone (e.g. https://ibm.github.io/the-mesh-for-data/v0.1/), however, most of the initial user information is here: https://ibm.github.io/the-mesh-for-data/v0.1/concepts/introduction/ which is a little hard to find.

**Comment by ronenkat** [comment]
hi @butler54 thank you for the feedback.
We have updated our webpage - https://mesh-for-data.github.io/mesh-for-data

Would you mind taking a look and give feedback whether you see things missing?
